### PR TITLE
[doc] Improve Pulsar Deployment DCOS

### PR DESCRIPTION
**Pull request by Monica-zy**
Motivation
Improve the language and the overall descriptive style of the Pulsar Security document (deploy-dcos section): http://pulsar.apache.org/docs/en/next/deploy-dcos/

Modifications
Adjust the tone, personal pronouns, voice also some typo errors of some sentences in the document.

### CODE REVIEW

**Comment by Monica-zy** [comment]
@Anonymitaet Please review this ^^

File: site2/docs/deploy-dcos.md [hunk 1]
```diff
@@ -42,14 +42,14 @@ You can deploy Pulsar on DC/OS using this command:
...
-This command will deploy Docker container instances in three groups, which together comprise a Pulsar cluster:
+This command deploys Docker container instances in three groups, which together comprise a Pulsar cluster:
 
 * 3 bookies (1 [bookie](reference-terminology.md#bookie) on each agent node and 1 [bookie recovery](http://bookkeeper.apache.org/docs/latest/admin/autorecovery/) instance)
 * 3 Pulsar [brokers](reference-terminology.md#broker) (1 broker on each node and 1 admin instance)
 * 1 [Prometheus](http://prometheus.io/) instance and 1 [Grafana](https://grafana.com/) instance
 
 
-> When running DC/OS, a ZooKeeper cluster is already running at `master.mesos:2181`, thus there's no need to install or start up ZooKeeper separately.
+> When you run DC/OS, a ZooKeeper cluster already run at `master.mesos:2181`, thus you do not have to install or start up ZooKeeper separately.
```
Comment (Anonymitaet) [inline site2/docs/deploy-dcos.md]:
```suggestion
> When you run DC/OS, a ZooKeeper cluster already runs at `master.mesos:2181`, thus you do not have to install or start up ZooKeeper separately.
```

File: site2/docs/deploy-dcos.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ Click into the instance of `prom` to get the endpoint of Prometheus, which is `1
 
 ![DC/OS prom endpoint](assets/dcos_prom_endpoint.png)
 
-If you click that endpoint, you'll see the Prometheus dashboard. The [http://192.168.65.121:9090/targets](http://192.168.65.121:9090/targets) URL will display all the bookies and brokers.
+If you click that endpoint, you see the Prometheus dashboard. The [http://192.168.65.121:9090/targets](http://192.168.65.121:9090/targets) URL display all the bookies and brokers.
```
Comment (Anonymitaet) [inline site2/docs/deploy-dcos.md]:
```suggestion
If you click that endpoint, you can see the Prometheus dashboard. The [http://192.168.65.121:9090/targets](http://192.168.65.121:9090/targets) URL display all the bookies and brokers.
```

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Pull request merged**